### PR TITLE
Reference customization in readme

### PR DESCRIPTION
**Pull request by XPhyro**
This PR adds a hacking section in the readme and documents #76 and #146.

### CODE REVIEW

**Comment by N-R-K** [comment]
I dont mind it, but I also feel like readme is getting a bit too crowded. Adding this to nsxiv-extra [CUSTOMIZATION](https://github.com/nsxiv/nsxiv-extra/blob/master/CUSTOMIZATION.md) and then linking it on the readme is also an option.

And now that I think about it, it seems like a better option since we already have some customization tips there. Adding it there would keep things unified.

**Comment by XPhyro** [comment]
I'll open a PR in nsxiv-extra and convert this one to reference [CUSTOMIZATION](https://github.com/nsxiv/nsxiv-extra/blob/master/CUSTOMIZATION.md) in the readme.

**Comment by N-R-K** [comment]
@XPhyro on nsxiv-extra, feel free to push directly. Also I would recommend using triple back-tick and then 'c' for codeblocks. It gives some nice syntax highlighting.
```c
int a = 5;
```

**Comment by XPhyro** [comment]
In this repo we seem to be using the indent syntax, so I used the same syntax. I'll do so in the future.

Right now there isn't much that could benefit from syntax highlighting in the readme and the contents of this PR will be in nsxiv-extra; but for future-proofing, should we update the readme to use backticks (in a different PR)? Using two different syntaxes would make reading the source file weird.

File: README.md [hunk 1]
```diff
@@ -161,16 +161,25 @@ Yes, see [nsxiv-env](https://github.com/nsxiv/nsxiv-extra/tree/master/scripts/ns
...
-suckless software, we host the [nsxiv-extra](https://github.com/nsxiv/nsxiv-extra)
-repo where users are free to submit whatever patches or scripts they wish.
+Due to our limited project scope, certain features or customization cannot be merged into
+nsxiv mainline. Following the spirit of suckless software, we host the
+[nsxiv-extra](https://github.com/nsxiv/nsxiv-extra) repo where users are free to submit
+whatever patches or scripts they wish.
 
 Description on how to use or submit patches can be found on
-[nsxiv-extra's](https://github.com/nsxiv/nsxiv-extra) README.
+nsxiv-extra's [README](https://github.com/nsxiv/nsxiv-extra/blob/master/README.md).
```
Comment (N-R-K) [inline README.md:182]: The repo already renders the README. Don't think we need to link to it specifically.
```suggestion
nsxiv-extra's [README](https://github.com/nsxiv/nsxiv-extra).
```

**Comment by XPhyro** [comment]
The nsxiv-extra commit is in: https://github.com/nsxiv/nsxiv-extra/commit/9aa41a44172ec4c995fd0e0cc7cead03c71620b3

Marking as ready for review.

File: README.md [hunk 2]
```diff
@@ -161,16 +161,25 @@ Yes, see [nsxiv-env](https://github.com/nsxiv/nsxiv-extra/tree/master/scripts/ns
...
+-------------
+
+The main method of customizing nsxiv is by setting values for the variables in *config.h*,
+or by using Xresources as explained in the manual. If these options are not sufficient,
+you may implement your own features by following
+[this guide](https://github.com/nsxiv/nsxiv-extra/blob/master/CUSTOMIZATION.md).
+
+If you think that your custom features can be beneficial for the general user base and
+is within our [project scope](CONTRIBUTING.md#Project-Scope), please submit it as a
+pull request on this repository, then we *may* merge it to mainline.
```
Comment (N-R-K) [inline README.md:174]: Think this paragraph should be below the paragraph about limited scope.

**Review by N-R-K — CHANGES_REQUESTED** [review]

**Review by N-R-K — APPROVED** [review]

**Pull request merged**